_includes/join-us/join-us-contribute-to-strimzi-band.html

```diff
@@ -0,0 +1,120 @@
+<div class="join-us-band">
+  <div class="component-slim grid-wrapper">
+    <div class="width-12-12">
+      <h2>Contribute to Strimzi</h2>
+
+      <p>Contributing to an open-source project is not just about writing code. 
+        There are many different ways you can contribute to Strimzi.</p>
+
+      <h3>Join the community</h3>
+
+      <p>You can start by joining the community. 
+        We have a <a href="https://slack.cncf.io/"><code>#strimzi</code> channel on the CNCF Slack workspace</a> which you can join and meet other Strimzi users. 
+        We also have <a href="https://lists.cncf.io/g/cncf-strimzi-users">user</a> and <a href="https://lists.cncf.io/g/cncf-strimzi-dev">developer</a> mailing lists. 
+        You can also come to our <a href="https://github.com/strimzi/strimzi-kafka-operator#strimzi-community-meetings">community calls</a> which happen every two weeks. 
+        On the community calls, you can learn about the current work or future plans. 
```

Comment:
It sounds like a kind of "unidirectional" meeting from maintainers to others. I would add even something like "raising questions, discussing issues you are facing or talking about your Strimzi use case". This will make it clear that users can be starters of discussions with maintainers.

Reply:
Better now?

Reply:
Yeah thanks!